src/test/java/org/cactoos/text/LengthOfTextTest.java

```diff
@@ -51,10 +51,26 @@ public void valueTextAsArg() throws Exception {
     @Test
     public void valueStringAsArg() throws Exception {
         MatcherAssert.assertThat(
+            new LengthOfText("string").value(),
+            Matchers.equalTo(6)
```

Comment:
@VsSekorin I'm sorry, but what do you think use `ScalarHasValue` instead of `Matchers.equalTo()`?